src/pullbacks/eig.jl

```diff
@@ -48,7 +48,8 @@ function eig_pullback!(
         Δgauge < gauge_atol ||
             @warn "`eig` cotangents sensitive to gauge choice: (|Δgauge| = $Δgauge)"
 
-        VᴴΔV .*= conj.(inv_safe.(transpose(D) .- D, degeneracy_atol))
+        VᴴΔV ./= conj.(transpose(D) .- D)
+        diagview(VᴴΔV) .= zero(eltype(VᴴΔV))
```

Comment:
leftover that can be removed?

Reply:
In that case we should probably update the tolerances instead?

Reply:
Let me see if I can find a good updated one that doesn't blow everything up

Reply:
OK, seems to be ok... Maybe again this was fixed inadvertently